tests/cron.test.ts

```diff
@@ -3,208 +3,190 @@ import sinon from 'sinon';
 import { CronJob, CronTime } from '../src';
 
 describe('cron', () => {
-	// eslint-disable-next-line jest/no-standalone-expect
-	afterEach(() => expect.hasAssertions());
+	let callback: jest.Mock;
+
+	beforeEach(() => {
+		callback = jest.fn();
+	});
+
+	afterAll(() => {
+		jest.clearAllMocks();
+	});
+
+	afterEach(() => sinon.restore());
 
 	describe('with seconds', () => {
 		it('should run every second (* * * * * *)', () => {
 			const clock = sinon.useFakeTimers();
-			const callback = jest.fn();
 			const job = new CronJob('* * * * * *', callback, null, true);
 
 			expect(callback).not.toHaveBeenCalled();
 			clock.tick(1000);
 			job.stop();
-			clock.restore();
 			expect(callback).toHaveBeenCalledTimes(1);
 		});
 
-		it('should run second with onComplete (* * * * * *)', done => {
+		it('should run second with onComplete (* * * * * *)', async () => {
```

Comment:
The `done` callback removed in favor of the async function must be reverted, in every test case. We mustn't introduce a breaking change, only a new feature, which should be opt-in.

Reply:
Apologies, I overlooked considering existing users. I have reverted that change and modified the code to pass all test cases.